rke/k8s_rke_system_images.go

```diff
@@ -6261,6 +6261,50 @@ func loadK8sRKESystemImages() map[string]v3.RKESystemImages {
 			WindowsPodInfraContainer:  "rancher/kubelet-pause:v0.1.6",
 			Nodelocal:                 "rancher/mirrored-k8s-dns-node-cache:1.15.13",
 		},
+		// Enabled in v2.6.4
+		"v1.20.14-rancher2-2": {
```

Comment:
Same as in 2.5 PR, I believe this should be `v1.20.14-rancher2-1`.

Reply:
`v1.20.14-rancher1-2` was there and it was merged 5 days ago so I just copied and used that template. Should I avoid `v1.20.14-rancher1-2` template? and make a copy from `v1.20.14-rancher1-1` ?

Reply:
@rayandas Yeah, I see where the confusion is coming from. We always copy from the previous template, so you've copied the right one `rancher1-2` but once we bump hyperkube, it's a new version so we start from `rancher2-1` again and not `rancher2-2`.